frontend/public/components/modals/create-namespace-modal.jsx

```diff
@@ -140,15 +141,36 @@ const CreateNamespaceModalWithTranslation = connect(
           className="modal-content modal-content--no-inner-scroll"
         >
           <ModalTitle>
-            {this.props.createProject ? t('public~Create Project') : t('public~Create Namespace')}
+            {createProject ? t('public~Create Project') : t('public~Create Namespace')}
           </ModalTitle>
           <ModalBody>
+            {createProject && FLAGS.OPENSHIFT === 'OPENSHIFT' ? (
```

Comment:
`FLAGS.OPENSHIFT === 'OPENSHIFT'` will always evaluate to `true`... You would need to use `connectToFlags` instead to get the flag. We don't actually need that check here, though, so I would leave it out. (If `createProject` is `true`, we already know it's an OpenShift cluster.)